### PR TITLE
Added shortcut information on Chrome, fixed German spelling mistake.

### PR DESCRIPTION
**Pull request by Ruudt**

### CODE REVIEW

**Comment by aschempp** [comment]
Thanks! Do you think it would make sense to create a table for the different browsers and platforms?

File: manual/en/02-Administration-area.md [hunk 1]
```diff
@@ -244,4 +244,5 @@ creating style sheets.</td>
 The keyboard shortcuts above only work on Windows in this form. Mac users have
 to use `[Ctrl]+[⌥ Opt]` instead of the `[Alt]` key together with the keyboard
 shortcut. In addition, Firefox on Windows deviates from the standard: you have
-to press `[Alt]+[Shift]` and the keyboard shortcut.
+to press `[Alt]+[Shift]` and the keyboard shortcut. Chrome users have to use 
+`[Alt]+[Shift]+e` and `[Alt]+[Shift]+f` for those shortcuts.
```
Comment (xchs) [inline manual/en/02-Administration-area.md]: The mentioned Chrome workaround applies only for Windows systems, right?
- Windows: https://support.google.com/chrome/answer/157179?hl=en
- Mac: https://support.google.com/chrome/answer/165450?hl=en
- Linux: https://support.google.com/chrome/answer/171571?hl=en

Comment (Ruudt) [inline manual/en/02-Administration-area.md]: Nope, honestly I thought about doing that and for reasons I don't understand didn't!
It should be added that this only applies to Windows and Linux.

**Comment by aschempp** [comment]
How about this?

<table>
<tr><th>Browser</th><th>Platform</th><th>Shortcut</th><th>Example</th></tr>
<tr><td>Firefox</td><td>Windows</td><td>Ctrl + Alt</td><td>Ctrl + Alt + C <i>(save & close)</td></tr>
<tr><td colspan="4">...</td></tr>
</table>

**Comment by Ruudt** [comment]
I would place the platform as first column because it is the dominant factor for keyboard layout. How would you add the two Chrome exceptions in a table? Asterisk with explanation at the bottom?

**Comment by xchs** [comment]
| Browser | Platform | Shortcut |
| --- | --- | --- |
| Firefox | Windows | [Alt]+[Shift] |
| Chrome | Windows, Linux | [Alt]+[Shift]+E, [Alt]+[Shift]+F |

**Comment by aschempp** [comment]
Maybe all Chrome shortcuts work with both keys pressed, too? So we could just "present" that as the regular option?

**Comment by Ruudt** [comment]
That is indeed the case; I tried `[Alt]+[Shift]+c` and it worked. However `[Alt]+[Shift]+t` is in use by Chrome as well (only on Windows). So less exceptions...

**Comment by aschempp** [comment]
Well it appears the shortcuts for Chrome on Windows are different than Mac and Linux. But they are the same for all accesskeys on the same browser and same platform.

**Comment by Ruudt** [comment]
Can't get it any neater then this:

**Shortcut exceptions**

| Platform | Browser | Shortcut |
| --- | --- | --- |
| Mac | All browsers | `[Ctrl]+[⌥ Opt]` |
| Windows | Firefox | `[Alt]+[Shift]` |
| Windows, Linux | Chrome | `[Alt]+[Shift]` and `[Alt]+t` instead of `[Alt]+[Shift]+t` |

**Pull request closed without merging**